### ISSUE TITLE
[BUG][Java Service][S3]Return to 520 when there is no component in requesting product/version on localelist and supportedLanguageList APIs. 

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#1927]
**Describe the bug**
commit: 5c86ed05f4c394a47afe25e7fc0501cdfa8f0bc8
Return to 520 when there is no component in requesting product/version on localelist and supportedLanguageList APIs. 
Reproduce on v1 and v2 apis:
GET [​/i18n​/api​/v1​/translation​/product​/{productName}​/version​/{version}​/supportedLocales](http://10.186.206.156:8091/i18n/api/doc/swagger-ui/index.html?urls.primaryName=v1#/translation-product-api/getSupportedLocalesUsingGET)
GET [​/i18n​/api​/v2​/locale​/supportedLanguageList](http://10.186.206.156:8091/i18n/api/doc/swagger-ui/index.html?urls.primaryName=v2#/locale-api/getDisplayLanguagesListUsingGET)
GET [​/i18n​/api​/v2​/translation​/products​/{productName}​/versions​/{version}​/localelist](http://10.186.206.156:8091/i18n/api/doc/swagger-ui/index.html?urls.primaryName=v2#/translation-product-api/getSupportedLocalesUsingGET_1)

**To Reproduce**
1. Go to 'GET [​/i18n​/api​/v2​/translation​/products​/{productName}​/versions​/{version}​/localelist]'
2. Request a product&version: no data under product/version folder.
3. Check the response:
```
{
  "response": {
    "code": 520,
    "message": "1",
    "serverTime": ""
  },
  "signature": "",
  "data": null
}
```

**Expected behavior**
Should return 200 and empty list.
```
{
  "response": {
    "code": 200,
    "message": "OK",
    "serverTime": ""
  },
  "signature": "",
  "data": {
    "locales": [],
    "version": "2.0.0",
    "productName": "SingleVersionNoData"
  }
}
```

**Comment by lyiyu66** [comment]
Same issue is reproducible on below apis:
GET
 [​/i18n​/api​/v2​/translation​/products​/{productName}​/versions​/{version}]
GET
 [​/i18n​/api​/v2​/translation​/products​/{productName}​/versions​/{version}​/locales​/{locale}​/components​/{component}]
GET
 [​/i18n​/api​/v2​/translation​/products​/{productName}​/versions​/{version}​/locales​/{locale}​/components​/{component}​/keys]
GET
 [​/i18n​/api​/v2​/translation​/products​/{productName}​/versions​/{version}​/locales​/{locale}​/components​/{component}​/keys​/{key}]

**Comment by tigershi** [comment]
the same issue with https://github.com/vmware/singleton/issues/1919

**Comment by lyiyu66** [comment]
Not same issue. Currently it return to 520.

**Comment by lyiyu66** [comment]
The issue is fixed with commit f1daa066efe95bce8dc732ee1fc1980d232094a9.
